sycl/doc/extensions/experimental/sycl_ext_codeplay_enqueue_native_command.asciidoc

```diff
@@ -173,13 +173,96 @@ dependencies are satisfied.
 The SYCL command described above completes once all of the native asynchronous
 tasks it contains have completed.
 
-The call to `interopCallable` must not submit any synchronous tasks to the
-native backend object, and it must not block waiting for any tasks to complete.
```

Comment:
@AerialMantis We need to make synchronous API calls to the backend API to add the native commands to the graph.

I'm not sure how to specify a restriction on some synchronous backend calls and not others without going into backend specifics which would be brittle wording.

What were the concerns with what a user would do in synchronous calls? If a user does some blocking work, then it blocks the `interopCallable` callable returning, which seems undesirable but not particularly ill defined.

Reply:
Updated to your suggested wording